### PR TITLE
Add "reportExplicit" endpoint

### PR DESCRIPTION
**Pull request by markspolakovs**
Untested, have fun

### CODE REVIEW

**Comment by URY-GitHub** [comment]
Thank you for this pull request, @markspolakovs!

@mstratford, would you mind reviewing this PR, please? Feel free to ask anyone for help if you're not quite sure what you're doing.

File: src/Classes/MyRadioEmail.php [hunk 1]
```diff
@@ -310,9 +310,9 @@ public static function sendEmailToUser(MyRadio_User $to, $subject, $message, MyR
     /**
      * Sends an email to the specified MyRadio_List.
      *
-     * @param MyRadio_List $to
-     * @param string       $subject email subject
-     * @param sting        $message email message
+     * @param MyRadio_List  $to
+     * @param string        $subject email subject
+     * @param string        $message email message
```
Comment (markspolakovs) [inline src/Classes/MyRadioEmail.php:315]: Didn't know Sting was moonlighting as a PHP type...

**Review by michael-grace — APPROVED** [review]
LGTM

**Review by mstratford — APPROVED** [review]
LGTM

**Pull request merged**